tools/prkit/verify.sh

```
@@ -0,0 +1,153 @@
+else
+  missing_skills=$(printf '%s\n' "$SKILL_REFS" | while read -r s; do [ -n "$s" ] && { [ -f "$P/skills/$s/SKILL.md" ] || echo "$s"; }; done)
+  if [ -n "$missing_skills" ]; then fail "ref-int: skills missing:"; printf '%s\n' "$missing_skills" | sed 's/^/         /'
+  else ok "ref-int: all invoked skills exist"; fi
+fi
+
+# --- 5. Out-of-set namespace gate: every prkit:<name> reference must resolve to a
+# shipped command/agent/skill OR an allowlisted label (`active`, `*-finding`).
+# Backstops the neutralizer's de-namespace: a NEW UberDev skill it didn't strip
+# would surface here as a dangling prkit:<name> and fail generation. ---
+SHIPPED="$(mktemp)"
+{ ls "$P/commands" 2>/dev/null | sed 's/\.md$//'
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)